Pod/libs/Headers/ADBMediaHeartbeat.h

```diff
@@ -0,0 +1,333 @@
+/*************************************************************************
+ *
+ * ADOBE CONFIDENTIAL
```

Comment:
@ladanazita should this be included in a public repo?

Reply:
They are eventually going to ship to cocoapods, so I'm not too worried — is it possible to have this still be a part of our integration pod but still keep it private?

Reply:
Not really. The header files at least have to be public.